source/search_page/assets/searchRecipe.js

```diff
@@ -52,25 +54,36 @@ class SearchRecipe extends HTMLElement {
           border-style: solid;
           border-width: 2px;
           border-color: black;
+          text-decoration: none; 
         }
 
         .recipe-time {
           color: white;
           margin-left:12px;
+          margin-top: -0.00005px
           text-align: center;
+          margin-bottom: 25px;
+          text-decoration: none; 
+          background-color: none;
+
+        }
         }
         `;
         
 
         styleElem.innerHTML = styles;
 
         const recipe = document.createElement('article');
-        recipe.setAttribute('href',"");
 
         const recipeTitle = document.createElement('p');
-        recipeTitle.setAttribute('href',"");
+        //recipeTitle.setAttribute('href', `../recipe-individual/inex.html?${data.id}`);
         recipeTitle.setAttribute('class','headline');
 
+        const link = document.createElement('a')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8435053)
```suggestion
        const link = document.createElement("a")
```